Source/SwiftLintFramework/Rules/RuleConfigurations/FileNameConfiguration.swift

```diff
@@ -6,10 +6,15 @@ public struct FileNameConfiguration: RuleConfiguration, Equatable {
 
     private(set) public var severity: SeverityConfiguration
     private(set) public var excluded: Set<String>
+    private(set) public var prefixPattern: String
+    private(set) public var suffixPattern: String
 
-    public init(severity: ViolationSeverity, excluded: [String] = []) {
+    public init(severity: ViolationSeverity, excluded: [String] = [],
+                prefixPattern: String = "", suffixPattern: String = "") {
```

Comment:
I'd keep the `suffixPattern` default value as `\\+.*` here just in case someone is using SwiftLintFramework directly.

Reply:
Sure, just added it back.